app/views/wishlists/show.html.erb

```diff
@@ -0,0 +1,15 @@
+<h1><%= @wishlist.name %></h1>
+
+<p><%= @wishlist.description %></p>
+
+<p>
+  <strong>Your unique link for this whislist:</strong><br>
+  <%= request.base_url %>/i/<%= @wishlist.token %>/
```

Comment:
To jest na prawdę zły pomysł tworzyć linki ręcznie. Wygeneruj się ścieżkę porządnie w `routes`:

    resources :invitations, param: :token, only: [:show]